src/components/calculator.jsx

```diff
@@ -1,35 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+import ResultView from './ResultView';
 
-const Calculator = () => (
-  <>
-    <div className="calculator">
-      <div className="result">0</div>
-      {/* symboles */}
-      <div>AC</div>
-      <div>+/-</div>
-      <div>%</div>
-      <div className="orange">÷</div>
-      {/* symboles */}
+const Calculator = () => {
+  const [obj, setObj] = useState(false);
```

Comment:
- [ ] There is an error in the browser console as a result of the prop type passed to `ResultView` component. You are creating state for the `obj` which is an object. Therefore, the default state cannot be a `boolean`, but rather can be an empty object like this 👇 
```javascript
const [obj, setObj] = useState({});
end
```
![image](https://github.com/Mahamadou-Nouridine/math-magicians/assets/46839548/88a68ae4-953b-4a3e-acd6-8358aedf2b3b)